src/main/cpp/_nix_based/jssc.cpp

```diff
@@ -49,6 +50,13 @@
 
 //#include <iostream> //-lCstd use for Solaris linker
 
+// TODO: IMHO logging to slf4j would be nice.
+#ifndef DEBUG
+    #define DEBUG 0
+#endif
+#define LOG_WARN( fmt, ... ) fprintf(stderr, "[WRN %s:%d] " fmt, strrchr(__FILE__,'/')+1, __LINE__,  __VA_ARGS__ )
```

Comment:
I've never seen the words "WARN" and "DEBUG" abbreviated.  I'm interested to know how standard this is.   Also, any debug capabilities should be consistent across platforms, so this should also go into the win32 stuff.

Reply:
Just a silly idea of mine. I think you're right and we should write them out 👍